src/lib_ccx/hardsubx_decoder.c

```diff
@@ -33,8 +34,8 @@ char* _process_frame_white_basic(struct lib_hardsubx_ctx *ctx, AVFrame *frame, i
 			int b=frame->data[0][p+2];
 			pixSetRGBPixel(im,j,i,r,g,b);
 			float L,A,B;
-			rgb_to_lab((float)r,(float)g,(float)b,&L,&A,&B);
-			if(L > ctx->lum_thresh)
+			rgb2lab((float)r,(float)g,(float)b,&L,&A,&B);
+			if(L > 85)
```

Comment:
This is an incorrect and unnecessary change. Please also do not change the naming convention of rgb_to_lab

Reply:
Yep, This is a mistake, I did it when I was testing, shouldn't have changed that in hardsubx_decoder.c